Not Compatible with latest libvips 8.7.0 release. failing to compile because of a duplicate declaration.

Hello,

The current version of bimg is incompatible with the latest version of libvips 8.7.0 and it fails to compile.

`vips_rotate` was added in v8.7.0 and is causing a compilation error due to duplicate declaration.

Compiler error:
```
# github.com/h2non/bimg
In file included from ../../go/pkg/mod/github.com/h2non/bimg@v1.0.18/vips.go:5:
./vips.h:180:1: error: conflicting types for 'vips_rotate'
vips_rotate(VipsImage *in, VipsImage **out, int angle) {
^
/usr/local/Cellar/vips/8.7.0/include/vips/resample.h:85:5: note: previous declaration is here
int vips_rotate( VipsImage *in, VipsImage **out, double angle, ... )
    ^
1 error generated.
```

--- a/preinstall.sh
+++ b/preinstall.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 vips_version_minimum=8.4.2
-vips_version_latest_major_minor=8.5
+vips_version_latest_major_minor=8.7
 vips_version_latest_patch=1
 vips_version_full="$vips_version_latest_major_minor.$vips_version_latest_patch"
 

--- a/vips.go
+++ b/vips.go
@@ -233,7 +233,7 @@ func vipsRotate(image *C.VipsImage, angle Angle) (*C.VipsImage, error) {
 	var out *C.VipsImage
 	defer C.g_object_unref(C.gpointer(image))
 
-	err := C.vips_rotate(image, &out, C.int(angle))
+	err := C.vips_rotate_bimg(image, &out, C.int(angle))
 	if err != 0 {
 		return nil, catchVipsError()
 	}

--- a/vips.h
+++ b/vips.h
@@ -177,7 +177,7 @@ vips_type_find_save_bridge(int t) {
 }
 
 int
-vips_rotate(VipsImage *in, VipsImage **out, int angle) {
+vips_rotate_bimg(VipsImage *in, VipsImage **out, int angle) {
 	int rotate = VIPS_ANGLE_D0;
 
 	angle %= 360;

